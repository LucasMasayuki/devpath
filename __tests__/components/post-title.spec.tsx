import React from 'react'
import renderer from 'react-test-renderer'
import PostTitle from '@/components/post-title'

describe('PostTitle', () => {
  it('should render PostTitle', () => {
    const component = renderer.create(
      <PostTitle>
        <div>title</div>
      </PostTitle>,
    )

    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
