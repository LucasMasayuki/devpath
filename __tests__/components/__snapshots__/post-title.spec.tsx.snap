// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostTitle should render PostTitle 1`] = `
<h1
  className="chakra-heading css-1dwxp6k"
>
  <div>
    title
  </div>
</h1>
`;
